bench/fastify-busboy-bench.js

```diff
@@ -0,0 +1,57 @@
+const Busboy = require('../lib/main');
+
+function createMultipartBuffer(boundary, size) {
+  const head =
+    '--' + boundary + '\r\n'
+    + 'content-disposition: form-data; name="field1"\r\n'
+    + '\r\n'
+    , tail = '\r\n--' + boundary + '--\r\n'
+    , buffer = Buffer.allocUnsafe(size);
+
+  buffer.write(head, 0, 'ascii');
+  buffer.write(tail, buffer.length - tail.length, 'ascii');
+  return buffer;
+}
+
+for (let i = 0, il = 10; i < il; i++) {
+  const boundary = '-----------------------------168072824752491622650073',
+    d = new Busboy({
+      headers: {
+        'content-type': 'multipart/form-data; boundary=' + boundary
+      }
+    }),
+    mb = 100,
+    buffer = createMultipartBuffer(boundary, mb * 1024 * 1024),
```

Comment:
we probably should extract shared part somewhere to reduce risk of benchmarks diverging over time.

Reply:
For Fastify-Busboy and Old-Busboy?

Reply:
at the very least, but I assume there is lots of duplication across all benchmarks (which we'll need revise eventually too).

Reply:
Could we do it in another PR? Benchmarks against other Projects has to be any way cleaned up.

Reply:
sure!